.travis/before_install.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Don't source setup.sh here, because the virtualenv might not be set up yet
+
+export NUMCORES=`grep -c ^processor /proc/cpuinfo`
+if [ ! -n "$NUMCORES" ]; then
+  export NUMCORES=`sysctl -n hw.ncpu`
+fi
+echo Using $NUMCORES cores
+
+echo ARST
+python --version
+
+# Install dependencies
+if [ "$TRAVIS_OS_NAME" == "linux" ]; then
+  sudo apt-get update
+  APT_INSTALL_CMD='sudo apt-get install -y --no-install-recommends'
+  $APT_INSTALL_CMD dos2unix
+
+  # Install protobuf
```

Comment:
nit: wrap this protobuf installation into a function

Reply:
ok, sure